drivers/cache/cache_nrf.c

```diff
@@ -68,14 +68,6 @@ static inline int _cache_all(NRF_CACHE_Type *cache, enum k_nrf_cache_op op)
 		return -ENOTSUP;
 	}
 
-	k_spinlock_key_t key = k_spin_lock(&lock);
-
-	/*
-	 * Invalidating the whole cache is dangerous. For good measure
-	 * disable the cache.
-	 */
-	nrf_cache_disable(cache);
```

Comment:
I understand the issue but invalidating the cache without disabling or flushing it means 100% crashing the kernel. Is this really something you want to expose to the user?

Reply:
invalidating of whole cache is not supported https://github.com/zephyrproject-rtos/zephyr/blob/8d762a0d2e1c9d0fa8eeabfe9f09cddf26754d37/drivers/cache/cache_nrf.c#L64